enterprise/frontend/src/embedding-sdk/components/public/index.ts

```diff
@@ -1,2 +1,5 @@
+import "ee-overrides";
```

Comment:
I don't think we necessarily need this for this PR anymore since I moved the SDK redux state to the `MetabaseProvider` (as shown above). But I think it's a good thing to have so we can test whitelabeling and things like that in the future.

Reply:
Ah, thanks for the explanation.